include/l3.h

```diff
@@ -69,13 +69,22 @@ int l3_init(const char *path);
  * the names of the log file and the executable.
  */
 #ifdef L3_LOC_ENABLED
-void l3__log_simple(loc_t loc, const char *msg,
+void l3__log_simple(const loc_t loc, const char *msg,
```

Comment:
Minor change done after you got the `.S` fast-logging interfaces working. Changed this to `const`.